docs/user_guide/trace.md

```diff
@@ -30,13 +30,57 @@
 
 Triton includes that capability to generate a detailed trace for
 individual inference requests. Tracing is enable by command-line
-arguments when running the tritonserver executable. For example,
+arguments when running the tritonserver executable.
+
+Starting in 23.04 release, Triton has one flag `--trace-config` to specify 
+global or trace mode specific configuration setting. The format of this flag 
+is `--tracing-config=<trace mode>,<setting>=<value>`, where `<trace mode>` 
+is either `triton` or `opentelemetry`. By default, trace mode is set to `triton`,
+and the server will use the Triton's trace APIs. For `opentelemetry`, 
+the server will use the [OpenTelemetry's APIs](#opentelemetry-trace-support) to generate, collect and export 
+traces for individual inference requests.
+
+To specify global trace settings (level, rate, count, or mode), 
+the format is `--tracing-config=<setting>=<value>`.
+
+An example ussage, which invokes Triton's trace APIs:
 
 ```
-$ tritonserver --trace-file=/tmp/trace.json --trace-rate=100 \
-    --trace-level=TIMESTAMPS --trace-log-frequency=50 --trace-count=100 ...
+$ tritonserver --trace-config=triton,file=/tmp/trace.json \
+    --trace-config=triton,log-frequency=50 --trace-config=rate=100 \
+    --trace-config=level=TIMESTAMPS --trace-config=triton,log-frequency=50 \
+    --trace-config=count=100 ...
 ```
 
+## Trace settings
+### Global settings
+The following table shows available global trace settings to pass to `--trace-config`
+
+| Setting     | Default Value |  Description                                                                                                                           |
+|:------------|:--------------|----------------------------------------------------------------------------------------------------------------------------------------|
+| `rate`      | 1000          | Specifies the sampling rate. The same as deprecated `--trace-rate`. In the above example every 100-th inference request will be traced.|
+| `level`     | OFF           | Indicates the level of trace detail that should be collected and may be specified multiple times to trace multiple informations. The same as deprecated `--trace-level`|
+| `count`     | -1            | Specifies the remaining number of traces to be collected. In the example Triton will stop tracing requests after 100 traces are collected. The same as deprecated `--trace-count`. |
+| `mode`      | triton        | Specifies which trace APIs to use for collecting traces. |
+
+### Triton trace APIs settings
```

Comment:
Can you add a similar `OpenTelemetry Trace API Settings` table that describes `url` and other relevant fields?

Reply:
It is on its way.